### ISSUE TITLE
Debugging ts-node?

### ISSUE THREAD

**Issue by fmj02** [Microsoft/nodejstools#1756]
![image](https://user-images.githubusercontent.com/3116731/32392384-c79b9c9e-c0d5-11e7-9ecd-02475337f0c5.png)

Results in: 

> Failed to start process.
Probable cause is the Node.js exe is corrupt, please re-install.

It fails because ts-node is not a binary, it is a script:
`@IF EXIST "%~dp0\node.exe" (
  "%~dp0\node.exe"  "%~dp0\..\ts-node\dist\bin.js" %*
) ELSE (
  @SETLOCAL
  @SET PATHEXT=%PATHEXT:;.JS;=;%
  node  "%~dp0\..\ts-node\dist\bin.js" %*
)`

After a minute I figured I can do this:

![image](https://user-images.githubusercontent.com/3116731/32392733-075a0f18-c0d7-11e7-9f0b-5f00716c5629.png)

But I'm not sure how debugging would work then.
![image](https://user-images.githubusercontent.com/3116731/32392968-0134e80a-c0d8-11e7-8d00-c0fca7604c65.png)

I think it won't make a sense to use ts-node after all.

**Comment by paulvanbrenk** [comment]
This is an interesting scenario.

Any reason you don't want to use the TypeScript compiler that ships with VS?

**Comment by fmj02** [comment]
It shipped with typeorm https://github.com/typeorm/typeorm that I use in my project I tried to open the boilerplate project with Visual Studio but it failed at running so I had to remove it to get it to work. I think there isn't any reason to use it, maybe because of the simplicity of invocation outside VS environment - you just write ts-node instead of tsc and then node. I think it just does it behind the scenes. I think it slows down the process anyway. I'm not a javascript guy.
